### PR TITLE
Migrate 'Cross-origin' from Sanity to MDX

### PR DESCRIPTION
**Pull request by alexisintech**
/docs/backend-requests/making/cross-origin

This PR

- migrates this page from Sanity to MDX
- adds links to respective references
- updates imports in code examples
- edits 'Clerk session tokens' title to 'Session tokens' to be search indexed more easily.
- fixes some grammatical-related issues on 'useAuth()' page

Acknowledging that these pages probably need work, and need their code examples updated - but for the scope of "Sanity to MDX Migration", this can be revisited at a later time, with better resources.

For reference, this is what the page looked like before this PR:

https://github.com/clerkinc/clerk-docs/assets/98043211/b5e3e302-667e-481f-ad8d-ca493d4006ef

### CODE REVIEW

File: docs/backend-requests/making/cross-origin.mdx [hunk 1]
```diff
@@ -1,5 +1,93 @@
 ---
-sanity_slug: /docs/request-authentication/cross-origin
+title: Cross-origin requests
+description: Learn about various ways to make cross-origin requests to the backend using the Clerk SDK.
 ---
 
-# This is a stub
+# Cross-origin requests
+
+If your client and server are on different origins (e.g. making an API call to a server on `api.foo.com` from JavaScript running on a client at `foo.com`), the [session token]() needs to be passed in a network request header. There are a few different ways this can be done on the front-end.
```
Comment (LekoArts) [inline docs/backend-requests/making/cross-origin.mdx:8]: `[session token]()` is missing a link here

File: docs/backend-requests/making/cross-origin.mdx [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+    return fetch(...args, {
+      headers: { Authorization: `Bearer ${await getToken()}` }
+    }).then(res => res.json());
+  };
+
+  return useSWR(url, fetcher);
+}
+```
+
+## Tanstack Query (FKA React Query)
```
Comment (LekoArts) [inline docs/backend-requests/making/cross-origin.mdx:51]: I didn't know the "FKA" acronym, maybe just drop it to convey the same thing?

File: docs/backend-requests/making/cross-origin.mdx [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+
+If you are using [Tanstack Query](https://tanstack.com/query/latest), formally known as React Query, it will follow a similar pattern composing the `useSession` hook.
```
Comment (LekoArts) [inline docs/backend-requests/making/cross-origin.mdx:53]: Similar pattern to what?

File: docs/backend-requests/making/cross-origin.mdx [hunk 4]
```diff
@@ -1,5 +1,93 @@
...
+
+<Callout type="warning">
+  Make sure that you have your application wrapped in `<QueryClientProvider />` with a `QueryClient` set.
```
Comment (LekoArts) [inline docs/backend-requests/making/cross-origin.mdx:56]: Might be worthwhile linking to official docs of Tanstack Query here

**Review by LekoArts — CHANGES_REQUESTED** [review]

**Review by BRKalow — APPROVED** [review]

**Pull request merged**